.github/workflows/ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         php:
           - 7.2
           - 7.4
+          - 8.0
+          - 8.1
```

Comment:
This won't work since PHP 8.0/8.1 are not supported by TYPO3v10/v9. Please move these two to separate `include` entries below.

Reply:
Can you pin point me to the correct syntax?

Reply:
Right where I mentioned: https://github.com/pagemachine/typo3-phinx/blob/953fe50c32a7d8191b48fa52501ef04b8356ac49/.github/workflows/ci.yml#L27

Reply:
I've committed your suggestions. Thanks.